app/templates/profile.html

```diff
@@ -1,249 +1,267 @@
+            <!-- End Profile card -->
+
+            <!-- Features card -->
+            <div class="card card-profile">
+                <div class="card-body">
+                    <p class="card-title">
+                        Features
+                    </p>
+                    <ul class="list-group">
+                        <li class="list-group-item list-group-item-action no-border">
+                            <a class="text-decoration-none text-secondary" href="#">Export my calendar</a>
+                        </li>
+
+                        <li class="list-group-item list-group-item-action no-border">
+                            <a class="text-decoration-none text-secondary" href="#">Your feature</a>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+            <!-- End Features card -->
+            <script src="../static/horoscope.js"></script>
+
+            <div class="card card-profile">
+                <div class="card-body">
+                    <p class="card-title">
+                        <div class="btn-group">
+                            <button type="button" class="btn btn-secondary dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                                daily horoscope
+                            </button>
+                            <div class="dropdown-menu dropdown-menu-right">
+                                <button onclick="daily_horoscope('aries')" class="dropdown-item" type="button">Aries</button>
+                                <button onclick="daily_horoscope('taurus')" class="dropdown-item" type="button">Taurus</button>
+                                <button onclick="daily_horoscope('gemini')" class="dropdown-item" type="button">Gemini</button>
+                                <button onclick="daily_horoscope('cancer')" class="dropdown-item" type="button">Cancer</button>
+                                <button onclick="daily_horoscope('leo')" class="dropdown-item" type="button">Leo</button>
+                                <button onclick="daily_horoscope('virgo')" class="dropdown-item" type="button">Virgo</button>
+                                <button onclick="daily_horoscope('libra')" class="dropdown-item" type="button">Libra</button>
+                                <button onclick="daily_horoscope('scorpio')" class="dropdown-item" type="button">Scorpio</button>
+                                <button onclick="daily_horoscope('sagittarius')" class="dropdown-item" type="button">Sagittarius</button>
+                                <button onclick="daily_horoscope('capricorn')" class="dropdown-item" type="button">Capricorn</button>
+                                <button onclick="daily_horoscope('aquarius')" class="dropdown-item" type="button">Aquarius</button>
+                                <button onclick="daily_horoscope('aquarius')" class="dropdown-item" type="button">Pisces</button>
```

Comment:
maybe filter it by the user birthday to only show his horoscope?

Reply:
That was the original idea, latter I decided that it will be better to give the option to see all the signs and not only the user so it's better my way :)